src/nrf_802154_swi.c

```diff
@@ -225,8 +227,19 @@ typedef struct
 
         struct
         {
-            bool * p_result;                              ///< CCA config update request result.
-        } cca_cfg_update;                                 ///< CCA config update request details.
+            bool * p_result; ///< CCA config update request result.
+        } cca_cfg_update;    ///< CCA config update request details.
+
+        struct
+        {
+            bool * p_result; ///< RSSI measurement request result.
+        } rssi_measure;      ///< RSSI measurement request details.
+
+        struct
+        {
+            int8_t * p_rssi;                              ///< RSSI measurement result.
+            bool   * p_result;                            ///< RSSI measurement status.
+        } rssi_get;                                       ///< RSSI measurement result getter details.
     } data;                                               ///< Request data depending on it's type.
```

Comment:
```suggestion
    } data;                                               ///< Request data depending on its type.
```
Serious punctuation issue.